roles/importer/files/importer/checkpointR8x/get_basic_config.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
```

Comment:
This file is remaining only for manual/standalone tests, isn't it?

Reply:
exactly